lib/fog/ovirt/compute.rb

```diff
@@ -79,7 +79,7 @@ module Shared
         def ovirt_attrs obj
           opts = {:raw => obj}
           obj.instance_variables.each do |v|
-            key = v.to_s.gsub("@","").to_sym
+            key = v.to_s.delete("@").to_sym
```

Comment:
Is this change related to the rubocop fix?

Reply:
See http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Performance/StringReplacement

Reply:
Thanks